honeybee_schema/model.py

```diff
@@ -5,72 +5,22 @@
 
 from ._base import IDdBaseModel, NoExtraBaseModel
 from .boundarycondition import Outdoors, Surface, Ground, Adiabatic
+
 from .energy.properties import ShadeEnergyPropertiesAbridged, \
     DoorEnergyPropertiesAbridged, ApertureEnergyPropertiesAbridged, \
     FaceEnergyPropertiesAbridged, RoomEnergyPropertiesAbridged, \
     ModelEnergyProperties
-from .radiance.properties import ShadeRadiancePropertiesAbridged, \
-    DoorRadiancePropertiesAbridged, ApertureRadiancePropertiesAbridged, \
-    FaceRadiancePropertiesAbridged, RoomRadiancePropertiesAbridged, \
-    ModelRadianceProperties
-
-
-class Plane(NoExtraBaseModel):
-
-    type: constr(regex='^Plane$') = 'Plane'
-
-    n: List[float] = Field(
-        ...,
-        description="Plane normal as 3 (x, y, z) values.",
-        min_items=3,
-        max_items=3
-    )
-
-    o: List[float] = Field(
-        ...,
-        description="Plane origin as 3 (x, y, z) values",
-        min_items=3,
-        max_items=3
-    )
-
-    x: List[float] = Field(
-        default=None,
-        description="Plane x-axis as 3 (x, y, z) values. If None, it is autocalculated.",
-        min_items=3,
-        max_items=3
-    )
-
-
-class Face3D(NoExtraBaseModel):
-    """A single planar face in 3D space."""
-
-    type: constr(regex='^Face3D$') = 'Face3D'
-
-    boundary: List[conlist(float, min_items=3, max_items=3)] = Field(
-        ...,
-        min_items=3,
-        description='A list of points representing the outer boundary vertices of '
-            'the face. The list should include at least 3 points and each point '
-            'should be a list of 3 (x, y, z) values.'
-    )
-
-    holes: List[conlist(conlist(float, min_items=3, max_items=3), min_items=3)] = Field(
-        default=None,
-        description='Optional list of lists with one list for each hole in the face.'
-            'Each hole should be a list of at least 3 points and each point a list '
-            'of 3 (x, y, z) values. If None, it will be assumed that there are no '
-            'holes in the face.'
-    )
-
-    plane: Plane = Field(
-        default=None,
-        description='Optional Plane indicating the plane in which the face exists.'
-            'If None, the plane will usually be derived from the boundary points.'
-    )
 
+from .geometry import Plane, Face3D
 
 class ShadePropertiesAbridged(BaseModel):
 
+    try:  # see if the module has already been imported
+        ShadeRadiancePropertiesAbridged
+    except NameError:
+        # import the module here (instead of at top) to avoid a circular import
+        from .radiance.properties import ShadeRadiancePropertiesAbridged
```

Comment:
Why do we still have this and the other try/except imports? Shouldn't we be able to import them on the top of the file?

Reply:
Yes, I missed these local imports in the model.py while fixing it. They're all at the top now.